plugins/invisible-typing/src/index.ts

```diff
@@ -28,7 +28,7 @@ export async function start(): Promise<void> {
       const globalInvisible = cfg.get("invisible", true);
       const channelWise = cfg.get("button", true) ? cfg.get("channelWise", true) : false;
       const channels = cfg.get("channels", { [channelId]: globalInvisible });
-      if (channelWise ? channels[channelId] : globalInvisible) {
+      if (channelWise ? channels[channelId] ?? globalInvisible : globalInvisible) {
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Consistent use of nullish coalescing operator across files.

Applying the same logic as in Icon.tsx here ensures consistency in how undefined channel settings are handled across the plugin.

```suggestion
if (channelWise ? (channels[channelId] ?? globalInvisible) : globalInvisible) {
```